graphics/tinygl/ztriangle.cpp

```diff
@@ -85,6 +102,10 @@ void FrameBuffer::putPixelTexture(int fbOffset, const TexelBuffer *texture,
 	if (kEnableScissor && scissorPixel(x + _a, y)) {
 		return;
 	}
+	if (kStippleEnabled && !applyStipplePattern(x + _a, y, _polygonStipplePattern)) {
```

Comment:
makes me wonder, does stipple exclude texture?

Reply:
This was removed, I can re-added later if necessary.